data/Person.csv

```diff
@@ -17,38 +17,38 @@ P0008,Lu,Zhongnan,en,male,yes,1969,,PL0022,1950,,,,,,,T00115,188,,,,,2017-07-05,
+P0017,Ma,Jia,en,male,yes,1969,1975,PL0008,1953,,Feng Huai,en,PL0002,,,T00009,219ff,http://www.chinawriter.com.cn/zxhy/member/9075.shtml,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
+P0017,马,佳,zh,male,yes,1969,1975,PL0008,1953,,冯淮 ,zh,PL0002,,,T00009,219ff,http://www.chinawriter.com.cn/zxhy/member/9075.shtml,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
+P0018,Yan,Li,en,male,no,NULL,NULL,NULL,1954,,,,PL0002,S03,,T00009,220,http://blog.sina.com.cn/s/blog_77f3ab7f0101aabk.html,ONLINE SOURCE,爷爷是上海名中医，叫严苍山 ,,2017-07-05,LH,2018-01-18,OS
+P0018,严,力,zh,male,no,NULL,NULL,NULL,1954,,,,PL0002,S03,,T00009,220,http://blog.sina.com.cn/s/blog_77f3ab7f0101aabk.html,ONLINE SOURCE,爷爷是上海名中医，叫严苍山 ,,2017-07-05,LH,2018-01-18,OS
 P0019,Guo,Lusheng,en,male,yes,1969,1971,PL0138,1948,,Shi Zhi,en,PL0137,,,T00009,229,T00011,74-75,author of the books Xiangxin weilai 相信未來 and Zhe shi sidian bafen Beijing 《這是四點零八分的北京》; both written in 1968. Very influential among zhiqing. ,,2017-07-05,LH,2018-01-16,OS
 P0019,郭,路生,zh,male,yes,1969,1971,PL0138,1948,,食指,zh,PL0137,,,T00009,229,T00011,74-75,author of the books Xiangxin weilai 相信未來 and Zhe shi sidian bafen Beijing 《這是四點零八分的北京》; both written in 1968. Very influential among zhiqing. ,,2017-07-05,LH,2018-01-16,OS
-P0020,Gao,Hua,en,male,,,,,1954,2011,,,,,Indirect,T00006,132-139,,,,,2017-07-05,LH,2017-12-13,MG
-P0020,高,华,zh,male,,,,,1954,2011,,,,,Indirect,T00006,132-139,,,,,2017-07-05,LH,2017-12-13,MG
-P0021,Guo,Shiying,en,male,,,,,1942,1968,,,,,Maybe Yes,T00011,75f 80,,,,,2017-07-05,LH,2017-11-17,MG
-P0021,郭,世英,zh,male,,,,,1942,1968,,,,,Maybe Yes,T00011,75f 80,,,,,2017-07-05,LH,2017-11-17,MG
-P0022,Zhang,Heci,en,male,,,,,1943,,,,,,Maybe Yes,T00011,75f 80,,,,,2017-07-05,LH,2017-11-17,MG
-P0022,张,鹤慈,zh,male,,,,,1943,,,,,,Maybe Yes,T00011,75f 80,,,,,2017-07-05,LH,2017-11-17,MG
+P0020,Gao,Hua,en,male,no,NULL,NULL,NULL,1954,2011,,,PL0090,S07,Indirect,T00006,132-139,https://zh.wikipedia.org/wiki/%E9%AB%98%E5%8D%8E,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
+P0020,高,华,zh,male,no,NULL,NULL,NULL,1954,2011,,,PL0090,S07,Indirect,T00006,132-139,https://zh.wikipedia.org/wiki/%E9%AB%98%E5%8D%8E,ONLINE SOURCE,,,2017-07-05,LH,2018-01-18,OS
+P0021,Guo,Shiying,en,male,no,NULL,NULL,NULL,1942,1968,,,PL0012,S03,yes,T00011,75f 80,https://baike.baidu.com/item/%E9%83%AD%E4%B8%96%E8%8B%B1/63658,ONLINE SOURCE,son of Guo Moruo and Yu Liqun ,,2017-07-05,LH,2018-01-18,OS
+P0021,郭,世英,zh,male,no,NULL,NULL,NULL,1942,1968,,,PL0012,S03,yes,T00011,75f 80,https://baike.baidu.com/item/%E9%83%AD%E4%B8%96%E8%8B%B1/63658,ONLINE SOURCE,son of Guo Moruo and Yu Liqun ,,2017-07-05,LH,2018-01-18,OS
+P0022,Zhang,Heci,en,male,no,NULL,NULL,NULL,1943,,,,,S03,yes,T00011,75f 80,精神重力與個人詞源 ——中國先鋒詩歌論 ,122,grandson of Zhang Dongsun 张东荪. Son of Zhang Zongbing 张宗炳. Was sent to a laogai for 15 years. He founded the X literary society and the X shouchao journal. ,pls add the source_2 in the text file,2017-07-05,LH,2018-01-18,OS
```

Comment:
If the grandparents or parents are in our DB i would use their ID, otherwise create an issue where we collect missing persons referenced in the DB but without a person entry.

Reply:
Yes, you are right. They should be included in our new kinship table. I just wanted to put the information somewhere first.